### DISCUSSION TITLE
Dealing with plasma fire

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#227] [Ideas]
Discussion for how Flock should deal with plasma fires. Discussion version of #161.

One way might be to just have a structure that can absorb plasma.

May be another way?

**Comment by amylizzle** [comment]
Since we added deconstruction, the flock do now sort-of have a way: they can open the walls to space. Obviously this isn't great, but it does reduce the urgency on the issue.

**Reply by TobleroneSwordfish** [reply]
Also of note is that flock are incentivized to immediately do this to any area they capture, since they don't need air and the crew do. While I'm not sure I like this too much, it's the same with blob, nukies etc.
